src/python/grpcio/grpc/__init__.py

```diff
@@ -858,6 +858,21 @@ def set_details(self, details):
     """
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    def abort(self, code, details):
+        """Terminates the current RPC with a non-OK by raising an exception
+    status code.
+
+    The code and details passed as arguments will supercede the existing ones.
+
+    Args:
+      code: A StatusCode object to be sent to the client. It should not be OK.
+      details: An arbitrary string to be sent to the client upon completion.
```

Comment:
It's not totally arbitrary; it has to be ASCIIifiable. There's probably a length limit but I have no idea what it might be.

Reply:
I copied this verbatim from other places it was used (concretely "set_details"). Let me know what you want it to say.

Reply:
How about "An ASCII-encodable string to be sent to the client upon termination of the RPC."?